deployment/hasura/metadata/actions.graphql

```diff
@@ -76,7 +76,13 @@ type Query {
 type Query {
   schedulingDslTypescript(
     missionModelId: Int!
-  ): SchedulingDslTypescriptResponse
+  ): DslTypescriptResponse
+}
+
+type Query {
+  constraintsDslTypescript(
+    missionModelId: ID!
```

Comment:
Is there a reason `ID!` is used here instead of `Int!`? We should probably be consistent across the different queries.

Reply:
Yep, I re-used the `hasuraMissionModelActionP` parser in merlin, which expects to deserialize the argument as a string. The existing action that already uses this parser is defined with `ID!` instead of `Int!`.

The query can accept an int as well though, so mechanically the UI wouldn't have to do anything different.

I can also change it to use `Int` because that's what it should be, but there's quite a few of them. Probably best to open a small follow-up PR later this week.

Reply:
Created MAPSS-1855 for this